deps/rabbitmq_codegen/amqp-rabbitmq-0.9.1.json

```diff
@@ -98,6 +98,118 @@
     ],
 
     "classes": [
+        {
+            "id": 60,
+            "methods": [{"id": 10,
```

Comment:
Just found out about this, nice find! 

@michaelklishin I was wondering would reordering the methods inside the basic also help improve it further?

The first one is qos, which is hardly the most often called one.
I assume it's something like deliver, publish, ack, consume(-ok), ...

Reply:
@ansd @michaelklishin My question probably got lost in the inbox, friendly ping =)

Reply:
@bollhals the same idea can be taken further, sure.